modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticMohrCoulombMulti.h

```diff
@@ -98,6 +98,9 @@ class TensorMechanicsPlasticMohrCoulombMulti : public TensorMechanicsPlasticMode
   /// Returns the model name (MohrCoulombMulti)
   virtual std::string modelName() const;
 
+  /// Returns _use_custom_returnMap
+  virtual bool useCustomReturnMap() const;
+
 
```

Comment:
got rid of lots of occurrences of extra wihitespaces.  obviously i like whitespace - easiest thing to read in the world :-)